app/controllers/application_controller.rb

```diff
@@ -230,6 +230,7 @@ def signed_in_url
     return reactivate_account_url if user_needs_to_reactivate_account?
     return url_for_pending_profile_reason if user_has_pending_profile?
     return backup_code_reminder_url if user_needs_backup_code_reminder?
+    return user_please_call_url if current_user.suspended?
```

Comment:
I'd move this earlier than `reactivate` because if a user is suspended we don't want to give them a chance to finish setting completing it, since that would be pointless